talktorials/5_compound_clustering/T5_compound_clustering.ipynb

```diff
@@ -142,7 +142,7 @@
        "        "
```

Comment:
<p>I am confused why the numbers change (new 626 clusters, before 593) but when I run the "original" notebook locally, I get those new numbers as well. </p><p> </p><p>The input did not change from my understanding, so the change is a bit of a mystery to me. @rhjohnstone and @jaimejgp, if you see something I do not, let me know.</p><p> </p><p>Otherwise, I suggest to simply update to those new numbers.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>